core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProjectOrchestrator.scala

```diff
@@ -184,10 +184,10 @@ class BillingProjectOrchestrator(ctx: RawlsRequestContext,
               )
             )
         }
-      billingProfileId <- billingRepository.getBillingProfileId(projectName)
-      projectLifecycle = billingProfileId match {
+      azureManagedAppCoordinates <- billingRepository.getAzureManagedAppCoordinates(projectName)
```

Comment:
Needed to change how we decide if the billing project is an Azure one. Note that we don't store a cloudPlatform.